app/assets/javascripts/components/ReportFilter.jsx

```diff
@@ -14,15 +14,15 @@ class ReportFilter extends React.Component {
       ? ReportFilter.getFilter('nt_rpm_threshold') : default_nt_rpm_threshold;
 
     const nt_zscore_start =
-      (nt_zscore_threshold.split('-').length > 0) ? parseInt(nt_zscore_threshold.split('-')[0], 10) : 0;
+      (nt_zscore_threshold.split(',').length > 0) ? parseInt(nt_zscore_threshold.split('-')[0], 10) : 0;
```

Comment:
I think you forgot to convert some '-' to ','.

Would you mind approving my pull request that specifically does just that, so we can have an unbroken master?   Since the work here needs one more iteration.